packages/ketcher-core/src/domain/entities/functionalGroup.ts

```diff
@@ -197,9 +197,10 @@ export class FunctionalGroup {
     const isAtomInSameFunctionalGroup = (otherAtomId, sgroup) =>
       sgroup.atoms.includes(otherAtomId)
     for (const sgroup of sgroups.values()) {
-      const isFunctionalGroup = FunctionalGroup.isFunctionalGroup(sgroup)
+      const isSGroup =
```

Comment:
isSGroup - added this condition to display the label of the super atom (invoked from reatom)